test/Menu-test.jsx

```diff
@@ -3,7 +3,7 @@ import sinon from 'sinon';
 import React from 'react/addons';
 import Menu from '../src/Menu';
 
```

Comment:
If this is changing to an import I'd prefer to remove the empty line --- keep the imports all bunched together.